addon/components/impagination-collection.js

```diff
@@ -28,6 +28,7 @@ export default Ember.Component.extend({
           Ember.run.debounce(this, 'flushQueue', offset, 1, false);
         }
       });
+      Object.defineProperty(collection, "length", { get: collection.length });
```

Comment:
does just passing in `length: datasetState.length` into `CollectionInterface.create` no make the nut?